### PR TITLE
groundstation DB integration

### PR DESCRIPTION
**Pull request by danielg0**

### CODE REVIEW

File: cmd/groundstation/main.go [hunk 1]
```diff
@@ -10,29 +10,28 @@ import (
 )
 
 func main() {
-	slog.Info("Starting groundstation")
```
Comment (aDotInTheVoid) [inline cmd/groundstation/main.go:13]: NIT: Theirs no reason to remove it. I'd like to have it log immediatly

File: cmd/groundstation/main.go [hunk 2]
```diff
@@ -10,29 +10,28 @@ import (
...
 	if err != nil {
 		slog.Error("when opening database", "err", err)
 		return
 	} else {
 		slog.Info("connected to database", "url", dbUrl)
 	}
 
-	slog.Info("Starting groundstation API server", "port", configuration.Server.Port)
+	// start servers
+	slog.Info("Starting groundstation dish server")
```
Comment (aDotInTheVoid) [inline cmd/groundstation/main.go:34]: NIT: wait, where did this terminology come from

**Review by aDotInTheVoid — APPROVED** [review]

**Comment by aDotInTheVoid** [comment]
If you feel like fixing the nit's that'd be nice, but it's late, so feel free to just not honestly.

File: cmd/groundstation/main.go (hunk 1 above)
Comment (danielg0) [reply cmd/groundstation/main.go:13]: Ah, I thought this message was indicating that the satellite->groundstation server was about to be started, so I moved it down to just before that actually occured, like how there's one for the api server

Comment (aDotInTheVoid) [reply cmd/groundstation/main.go:13]: No, it's that the *binary* has started. Their can be a seperate line for the server

Comment (danielg0) [reply cmd/groundstation/main.go:13]: Cool, will add back

File: cmd/groundstation/main.go (hunk 2 above)
Comment (danielg0) [reply cmd/groundstation/main.go:34]: The API server had a name (api) and a message indicating it had booted, so I thought the satellite->groundstation could use one too.
Happy to take suggestions on what we call the communication from satellite->groundstation, but I personally do quite like dish (it's like the reciever/transmitter to satellites on a groundstation)

Comment (danielg0) [reply cmd/groundstation/main.go:34]: Alternatively, we could make the "groundstation api server", just "api server" and make the satellite->groundstation server the "groundstation server"

Comment (aDotInTheVoid) [reply cmd/groundstation/main.go:34]: Can we bikeshed this another day please!!

Comment (danielg0) [reply cmd/groundstation/main.go:34]: Of course. I'm sure we'll have a wheely great time

**Pull request merged**